src/node/rpc/managementfrontend.h

```diff
@@ -93,10 +93,18 @@ namespace ccf
         }
       };
 
+      auto get_quotes = [&node](RequestArgs& args) {
```

Comment:
Should this RPC also take the index of a node as argument so that we can only query a specific node if necessary?

Reply:
We could, but it's ftrivial tp extract out of the resulting JSON. I expect the serialisation/deserialisation costs to be quite negligible compared to the RPC overhead, and this is going to be a very infrequent request, so I'm inclined to go towards less code here, and only getQuotes, unless there's an argument I've missed for getQuote.

What I think is probably useful is to return the node certs as well as the quotes, because the node ids won't be very meaningful to users.

Reply:
@jumaffre thoughts?

Reply:
> What I think is probably useful is to return the node certs as well as the quotes, because the node ids won't be very meaningful to users.

Good point - we should definitely return those as well. 

I'm happy to only have `getQuotes` that returns the quotes for all nodes in the network.